src/process/native/win32.js

```diff
@@ -1,9 +1,10 @@
 import { exec } from 'child_process';
 
-export const getProcesses = () => new Promise(res => exec(`wmic process get ProcessID,ExecutablePath /format:csv`, (e, out) => {
+export const getProcesses = () => new Promise(res => exec(`wmic process get ProcessID,ExecutablePath,Name /format:csv`, (e, out) => {
   res(out.toString().split('\r\n').slice(2).map(x => {
     const parsed = x.trim().split(',').slice(1).reverse();
     parsed[0] = parseInt(parsed[0]) || parsed[0]; // pid to int
-    return parsed;
+    parsed[1] = parsed[2] || parsed[1]
```

Comment:
Use `return [ parseInt(parsed[0]) || parsed[0], parsed[1] ?? parsed[2] ];` instead

Reply:
should be `return [ parseInt(parsed[0]) || parsed[0], parsed[2] || parsed[1] ];`

Reply:
Nvm wmic lol.